### DISCUSSION TITLE
How to change theme photos and background images if I am using my new theme based on its parent theme - Breeze Blank?

### DISCUSSION THREAD

**Discussion by ghost** [breezefront/community#2] [1. Help]
Hi. I created a new theme based on parent theme "Breeze Blank". How can I edit the html body and stop using your theme's photos and logos by editing documents in /var/www/html/ for my new theme permanently?

Thank you Support Team!

**Comment by vovayatsyuk** [comment]
Hi! You can place your logo in `web/images/logo.svg` in your theme and it will be applied automatically unless it's overridden in the backend configuration.

How it's done in blank theme: https://github.com/breezefront/theme-frontend-breeze-blank/tree/master/web/images

**Reply by ghost** [reply]
How about background image and `~` at the footer?

**Reply by vovayatsyuk** [reply]
Wave and blob images are the part of evolution theme.

Blob: https://github.com/breezefront/theme-frontend-breeze-evolution/blob/master/web/css/theme/abstracts/_variables.less#L271
Footer: https://github.com/breezefront/theme-frontend-breeze-evolution/blob/master/web/css/theme/abstracts/_variables.less#L74
